bootcamp/tutorials/milvus_usages/funnel_search_with_matryoshka.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    dfs[0]
</pre><p>remove this debugging log</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>